Game.cpp

```diff
@@ -168,23 +185,52 @@ void Game::processInput()
                         break;
                     case SDL_SCANCODE_DELETE:
                         loadNewShape();
-                        break;
+                        continue;
+                    default:
+                        continue;
                 }
-                prepareScene();
-                std::cout << evnt.key.keysym.scancode << std::endl;
-
+                lockPiece();
```

Comment:
I added a lock when the player handles a piece. 
https://tetris.wiki/Lock_delay
> Some games such as the [Puyo series](https://tetris.wiki/Puyo_Pop) use a fixed amount of lock delay for each piece, pausing the timer while the piece falls, resetting it only when a new piece enters (entry reset). Some games, especially those using [Sega rotation](https://tetris.wiki/Sega_rotation) or its successors, reset the timer every time the piece "steps" downward (step reset). Games following the [Tetris Guideline](https://tetris.wiki/Tetris_Guideline) generally reset the timer on any successful shifting or rotation (move reset). There is usually a limit on the number of resets before a "step" downward; older guideline games had [Infinity](https://tetris.wiki/Infinity) where there was no limit.